backend/endpoints_rooms.go

```diff
@@ -225,6 +225,11 @@ type PingPongMessageBi struct {
 	Timestamp int    `json:"timestamp"`
 }
 
+type TypingIndicatorMessageBi struct {
+	Type string `json:"type"` 
+	Data map[string]bool `json:"data"`
```

Comment:
This is definitely a valid approach to implementing typing indicators, but I feel like this has multiple issues

I think it would be better if clients sent a `timestamp` of when they started typing, and other clients receive `userID` + `timestamp`

I address what that approach fixes in my other review comments